1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.ts

```diff
@@ -0,0 +1,21 @@
+export const InvalidLengthError = new Error('Password must be between 5 and 15 characters');
+
+export const InvalidUpperCaseError = new Error('Password must contain at least one uppercase letter');
+
+export const InvalidDigitError = new Error('Password must contain at least one digit');
+
+export interface ResponseObject {
+    valid: boolean;
+    errors: PasswordError[];
+}
+
+type PasswordError = typeof InvalidLengthError | typeof InvalidUpperCaseError | typeof InvalidDigitError;
+
+export class PasswordValidator {
```

Comment:
It looks like you've jumped to doing the _Obvious Implementation_ here. I remember this from your previous submissions, this is definitely one of those things where if you have a high level of skill and comfort with your ability to cover your tracks, yeah definitely go ahead and do this. 

What am I referring to here?

You've got _InvalidLengthError_, _UpperCaseError_ and _InvalidDigitError_ but the test responsibility spec is only about the password being too short. 

Definitely want to make sure you cover those responsibilities clearly and have them defined moving forward, because we're a bit out of sequence here.